### ISSUE TITLE
Local file not loaded

### ISSUE THREAD

**Issue by cnicodeme** [innoveit/play2-pdf#3]
Hi,

when callign a file from local (instead of absolute url), like CSS or Images, the PdfGenerator fails with the error : 

```
java.net.MalformedURLException: no protocol: /public/css/general.css
```

But

`System.out.println(Play.current().resourceAsStream("/public/css/general.css"));` correctly returns true.

**Comment by marcosinigaglia** [comment]
Hi, can you try the 1.1.2-SNAPSHOT version?
Thanks, Marco

**Comment by marcosinigaglia** [comment]
Fixed in 1.1.2
